Extensions/SonarQube/license-terms.md

```diff
@@ -1,5 +1,5 @@
 The MIT License (MIT)
```

Comment:
You will need to double check with @tomverin or Olivier but I think this can also be moved to LGPL.